addons/hr_recruitment/migrations/13.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+_column_renames = {
+    'hr_recruitment_stage': [
+        ('job_id', None),
+    ],
+}
+
+
+def alter_hr_applicant_email_cc_type(cr):
+    openupgrade.logged_query(
+        cr, """
+        ALTER TABLE hr_applicant
+        ALTER COLUMN email_cc
+        TYPE varchar
```

Comment:
You can see here that casting Text fields to Char is supported by Odoo natively
https://github.com/odoo/odoo/blob/13.0/odoo/fields.py#L1546-L1547

Reply:
Removed and documentation updated